app/car.py

```diff
@@ -0,0 +1,16 @@
+from decimal import Decimal
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: Decimal) -> None:
+        self.brand = brand
+        self.fuel_consumption = Decimal(fuel_consumption)
+
+    def count_fuel_consumption(
+        self, fuel_price: Decimal, length_way: Decimal
+    ) -> Decimal:
+        return (
+            ((Decimal(length_way) / 100) * self.fuel_consumption)
+            * Decimal(fuel_price)
```

Comment:
length_way, fuel_price - must be a Decimal from your function, why you transform it again?